driver-core/src/resources/META-INF/native-image/org.mongodb/mongodb-driver-core/native-image.properties

```diff
@@ -13,4 +13,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-Args = --initialize-at-run-time=com.mongodb.UnixServerAddress,com.mongodb.internal.connection.SnappyCompressor
```

Comment:
I saw that we had JAVA-3580 open to test UnixServerAddress and SnappyCompressor, which were proposed in #527. I do not know how we might test these, since the original had no reproducer. I do not understand why these were originally added, since they seem to never have had statics (initialized via `--initialize-at-build-time`). Either these were never needed, or there is some other form of initialization we should check our codebase for. Any ideas?

In any case, my testing shows that classes are being picked up from `native-image.properties`, so I think we should consider these sufficiently tested.